src/analysisd/ar_json.c

```diff
@@ -0,0 +1,80 @@
+/* Copyright (C) 2015-2021, Wazuh Inc.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+*/
+
+#include "config.h"
+#include "format/to_json.h"
+
+#define VERSION "1.0"
+#ifndef ARGV0
+#define ARGV0 "wazuh-analysisd"
+#endif
+
+/**
+ * @brief Build the JSON message
+ *
+ * @param[in] lf Event information.
+ * @param[in] ar Active Response information.
+ * @param[out] temp_msg Message in JSON format.
+ * @return int, 0 on success or 1 on failure.
+ */
+int getActiveResponseInJSON(const Eventinfo *lf, const active_response *ar, char *temp_msg)
+{
+    cJSON *_object = NULL;
+    cJSON *_array = NULL;
+    cJSON *json_alert = NULL;
+
+    cJSON *message = cJSON_CreateObject();
+    if (message == NULL){
+        merror("Failed to create active response JSON");
+        return 1;
+    }
+
+    cJSON_AddStringToObject(message, "version", VERSION);
+    
+    _object = cJSON_CreateObject();
+    cJSON_AddItemToObject(message, "origin", _object);
+    cJSON_AddStringToObject(_object, "name", ar->name);
```

Comment:
This is not correct. This should be the name of the node that generates the AR (manager node name).

Take a look how you can get it [here](https://github.com/wazuh/wazuh/blob/master/src/wazuh_modules/agent_upgrade/manager/wm_agent_upgrade_parsing.c#L413).